packages/sakuli-core/src/forwarder/connect-forwarder-to-test-execution-context.function.spec.ts

```diff
@@ -115,7 +115,12 @@ describe("connectForwarderToTestExecutionContext", () => {
       await rejectionTeardown();
```

Comment:
The test is sucessfull but prints some warnings to the console.

```
(node:22693) UnhandledPromiseRejectionWarning: Error: forwarder does not forward...
(Use `node --trace-warnings ...` to show where the warning was created)
(node:22693) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:22693) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

maybe we should address this.